composer.json

```diff
@@ -18,5 +18,11 @@
     "phpdocumentor/reflection-common": "^1.0@dev",
     "phpdocumentor/reflection-docblock": "^3.0@dev",
     "sebastian/diff": "^1.3"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "^6.0"
+  },
+  "scripts": {
```

Comment:
Adding the Composer script allows to easily run the correct version of PHPUnit (but also other tools such as PHPCS, should they be added in the future) without installing anything globally or modifying the PATH. See: https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands
Do you want me to drop the script? (In that case, would `.travis.yml` require the use of `vendor/bin/phpunit` to work with PHPUnit 6? I'm not sure)